app/frontend/src/loginContext.tsx

```diff
@@ -0,0 +1,5 @@
+import { createContext } from "react";
```

Comment:
Can you add a docstring to this file to explain how to use it? I'm actually new to React contexts (in my days, we didnt have contexts..) so an explanation would be helpful.